src/test/java/org/kohsuke/github/GHRepositoryForkBuilderTest.java

```diff
@@ -0,0 +1,277 @@
+package org.kohsuke.github;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InterruptedIOException;
+import java.time.Duration;
+import java.util.Map;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.*;
+
+// TODO: Auto-generated Javadoc
+
+/**
+ * The Class GHRepositoryForkBuilderTest.
+ */
+public class GHRepositoryForkBuilderTest extends AbstractGitHubWireMockTest {
+    private GHRepository repo;
+    private static final String TARGET_ORG = "nts-api-test-org";
+    private int originalInterval;
+
+    /**
+     * Instantiates a new Gh repository fork builder test.
+     */
+    public GHRepositoryForkBuilderTest() {
+    }
+
+    /**
+     * Sets up.
+     *
+     * @throws Exception
+     *             the exception
+     */
+    @Before
+    public void setUp() throws Exception {
+        repo = getTempRepository();
+
+        originalInterval = GHRepositoryForkBuilder.FORK_RETRY_INTERVAL;
+        GHRepositoryForkBuilder.FORK_RETRY_INTERVAL = 100;
+
+        String defaultBranch = repo.getDefaultBranch();
```

Comment:
Sorry, about this.  
Could commit this, delete your test JSON files, and rerecord.   It will significantly reduce the number of test JSON files.  We don't need to records the creation of three branches as part of every test. 

```suggestion
        GitHub github = getNonRecordingGitHub();
        GHRepository repo = github.getRepository(this.repo.getFullName());
        String defaultBranch = repo.getDefaultBranch();
```